lib/task-data/tasks/install-esx.js

```diff
@@ -17,6 +17,9 @@ module.exports = {
         esxBootConfigTemplateUri: '{{ api.templates }}/{{ options.esxBootConfigTemplate }}?nodeId={{ task.nodeId }}', //jshint ignore: line
         comport: 'com1',
         comportaddress: '0x3f8', //com1=0x3f8, com2=0x2f8, com3=0x3e8, com4=0x2e8
+        gdbPort: 'default',
+        logPort: 'com1',
```

Comment:
comport is the device name of UART in ESXi. comportaddress is the physical address of UART. logport is the port that ESXi throw out debug log during boot/running. There are different. I think they couldn't get merged